ffi.c

```diff
@@ -435,7 +435,21 @@ EXP K deref(K x) {
   return r;
 }
 
-#define N 9
+Z K1(cv){
```

Comment:
rename to`cvar`

Reply:
Are you sure it is necessary to export functions that are exposed via `dl`? I don't know much about Windows, but Python uses a similar mechanism to export functions from C modules and they use addresses of static functions to populate method lookup tables.